api/src/pcapi/core/external/attributes/models.py

```diff
@@ -91,6 +91,7 @@ class ProAttributes:
     has_banner_url: bool | None = (
         None  # Set to False when at least one permanent venue doesn't have a banner URL, True otherwise
     )
+    eac_meg: bool | None = None  # At least one collective offer with 'Marseille en Grand'
```

Comment:
Un `is` ou `has` dans le nom de l'attribut serait plus cohérent avec le nommage des autres booléens.

Reply:
Ok